specification/src/main/asciidoc/platform/ApplicationProgrammingInterface.adoc

```diff
@@ -1015,6 +1060,8 @@ _http://www.omg.org/cgi-bin/doc?formal/2000-06-19_ .
 
 ===== RMI-JRMP
 
+*TODO Note:* What to do with this section? Leave as-is, mark Optional, remove?
```

Comment:
I think there's a deeper question here because the actual RMI protocol has to my knowledge never been a requirement and I can guarantee is not tested in the TCK.  Geronimo never supported the actual RMI protocol and passed EE 4, 5 and 6 full profile TCKs.  I don't think even GlassFish supports EJB calls over RMI, just CORBA.

Reply:
We'll need to continue the discussion you started on the Platform mailing list.